examples/python/live-recognition/detect_barcodes.py

```diff
@@ -56,59 +60,68 @@ def recognize_live(recognizer:scanbotsdk.BarcodeRecognizer, device:str, use_disp
+            cv.namedWindow("BarcodeRecognizer", cv.WINDOW_AUTOSIZE)
+        image = np.zeros((DEVICE_CAPTURE_HEIGHT, DEVICE_CAPTURE_WIDTH, 3), dtype=np.uint8)
+        while True:
+            # Note that the image is updated in-place
+            ret, _ = cap.read(image)
+            if not ret:
+                print("Failed to capture image")
                 break
-                    
-    cap.release()
+
+            # Although it is not required to use the image ref as a context manager, it is recommended to do so when
+            # creating from live source to ensure that the image_ref is timely released.
+            #
+            # !!! IMPORTANT !!! It is critically important to specify the live_source=True option when creating the
+            # image ref to run the scanner in live mode. As an alternative it is possible to specify
+            # processing_mode=ProcessingMode.LIVE in the scanner configuration.
+            with scanbotsdk.ImageRef.from_ndarray(image, scanbotsdk.RawImageLoadOptions(live_source=True)) as image_ref:
+                barcode_scanning_result = scanner.run(
+                    image=image_ref)
+                parse_barcodes(barcode_scanning_result.barcodes, image)
+
+                if use_display:
+                    width = image.shape[1] * DISPLAY_HEIGHT / image.shape[0]
+                    to_show = cv.resize(image, (int(width), DISPLAY_HEIGHT))
+                    cv.imshow("BarcodeRecognizer", to_show)
+                    cv.waitKey(1)
+
+                    if cv.getWindowProperty("BarcodeRecognizer", cv.WND_PROP_AUTOSIZE) == -1:
+                        break
+    finally:
+        cap.release()
 
 
 def recognize():
     parser = argparse.ArgumentParser(description='Detect barcodes in an image.')
-    parser.add_argument('--input', type=str, required=True, help='Input device name (jetson_csi|libcamera) or number of /dev/video* device')
+    parser.add_argument('--input', type=str, required=True,
+                        help='Input device name (jetson_csi|libcamera) or number of /dev/video* device')
     parser.add_argument('--use-display', action='store_true', help='Show results in a window')
     parser.add_argument('--use-tensorrt', action='store_true', help='Use TensorRT backend for GPU acceleration')
-    
 
     args = parser.parse_args()
     input_device = args.input
     use_display = args.use_display
-    
+
     # Setup and initialize the Scanbot SDK
     print(f"Initializing Scanbot SDK...")
     scanbotsdk.initialize(LICENSE_KEY)
     print(f"License Status: {scanbotsdk.get_license_status()}")
 
-    # Setup recognition parameters
-    params = scanbotsdk.BarcodeRecognizerInitParams(
-        engine_mode=scanbotsdk.BarcodeRecognitionEngineMode.FAST,
-    )
-
+    # Setup scanning configuration
+    configuration = scanbotsdk.BarcodeScannerConfiguration(
+        barcode_format_configurations=[
```

Comment:
Explicitly add `processing_mode=ProcessingMode.AUTO` or `LIVE` here as an example, so that people know where to look for it.